tracer/src/Datadog.Trace.Tools.dd_dotnet/Checks/ProcessBasicCheck.cs

```diff
@@ -181,32 +207,41 @@ public static bool Run(ProcessInfo process, IRegistryService? registryService =
             // Running non-blocker checks after confirming setup was done correctly
             if (ok)
             {
+                AnsiConsole.WriteLine();
+                AnsiConsole.WriteLine(ConfigurationChecks);
+                AnsiConsole.WriteLine(TraceEnabledCheck("1"));
```

Comment:
If you went with my variation, You would call `ResetChecks();` at the start of this block.

Alternatively, you could make `ConfigurationChecks` a method, and do the reset as part of that call 🙂